admin/plugins.php

```diff
@@ -64,7 +81,8 @@ function perflab_render_plugins_ui() {
 	require_once ABSPATH . 'wp-admin/includes/plugin-install.php';
 	require_once ABSPATH . 'wp-admin/includes/plugin.php';
 
-	$standalone_plugins = array();
+	$standalone_plugins                         = array();
+	$get_active_modules_with_standalone_plugins = perflab_get_active_modules_with_standalone_plugins();
```

Comment:
Please revert, as this isn't used anymore.
```suggestion
	$standalone_plugins = array();
```